content/en/blogs/install-kubernetes-containerd-multus.md

```diff
@@ -0,0 +1,259 @@
+---
+title: 'Install Kubernetes 1.23, containerd, and Multus CNI the Easy Way'
+tag: 'Kubernetes, KubeKey'
+keywords: 'Kubernetes, containerd, docker, Multus CNI, '
+description: 'Install Kubernetes 1.23, containerd, and Multus CNI in a Linux machine within minutes.'
+createTime: '2021-12-26'
+author: 'Feynman'
+snapshot: '/images/blogs/en/kubekey-containerd/kubernetes-containerd-banner.png'
+---
+
+![k8s-containerd](/images/blogs/en/kubekey-containerd/kubernetes-containerd-banner.png)
+
+[KubeKey](https://github.com/kubesphere/kubekey) is a lightweight and turn-key installer that supports the installation of Kubernetes, KubeSphere and related add-ons. Writtent in Go, KubeKey enables you to set up a Kubernetes cluster within minutes. 
+
+Kubernetes 1.23 [was released on Dec 7](https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/). KubeKey has supported the installation of the latest version Kubernetes in its v2.0.0 alpha release, it also brought some new features such as support Multus CNI, Feature Gates, and easy-to-use air-gapped installation, etc. 
```

Comment:
`release, it also` -> `release, and also`

`support Multus` -> `support for Multus`